src/views/Requests.svelte

```diff
@@ -58,7 +59,10 @@ function viewToggled(choice) {
         {/each}
       {/if}
     {:else}
-      <div class="col-12 my-2 mx-5"><i class="text-muted">None found</i></div>
+      <div class="col-12 my-2 mx-5">
```

Comment:
I'd like to do a little more here for the user...seems redundant and a bit cluttered in the screenshot to me.  Maybe more content in the case of none here...

Reply:
I toyed with some verbiage, but didn't like anything I came up with. Let's talk.